assets/js/blocks/legacy-template/README.md

```diff
@@ -0,0 +1,18 @@
+# Legacy Template Block
+
+The Legacy Template block is a placeholder block for specific WooCommerce block templates which are rendered on the server-side when a block theme is active.
+
+Given a specific template attribute prop, the block will render a specific template both on the front-end and in the Site Editor in the form of a placeholder.
+
```

Comment:
>I'm wondering if here we should add a paragraph explaining that the placeholder does not completely represent what is rendered on the front-end.

It cannot hurt to add a paragraph explaining that so no problem, it's probably more useful to belong in the user/merchant facing docs but either way not a problem.

>this might be useful so that people are aware that this is not going to happen yet and don't get surprised when it doesn't.

Personally, I don't think this really belongs in these docs. Once a technical decision & change is made we should also update the documentation accordingly. I'd prefer to avoid adding "X or Y might be coming in the future" as it can lead to some confusion and muddy the waters for those who are looking for what is implemented right now.

We can however refer to other documentation, or a roadmap where this might be more applicable?

Reply:
Yea, I just mean we should warn people that styles and other customizations won't be applied to the placeholder. I'm not suggesting we necessarily write in the docs that it's our intention to make that happen eventually.

(Though, to be fair, we have some reference to future plans [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5939/files#diff-162dd54970925f7e59d7d2b0fdda4785f6b4ed538b628459e4a89bf2f7767952R43), though I think it's different in this case since that's an over-arching goal and not a detail such as styles applied to the placeholder, so I'd be for leaving that reference to the long-term blockification into the docs, personally)